sdk/healthinsights/azure-health-insights-radiologyinsights/README.md

```diff
@@ -56,9 +56,12 @@ Build a **synchronous** client:
 String endpoint = Configuration.getGlobalConfiguration().get("AZURE_HEALTH_INSIGHTS_ENDPOINT");
 String apiKey = Configuration.getGlobalConfiguration().get("AZURE_HEALTH_INSIGHTS_API_KEY");
 
-RadiologyInsightsClient radiologyInsightsClient = new RadiologyInsightsClientBuilder()
-        .endpoint(endpoint).serviceVersion(RadiologyInsightsServiceVersion.getLatest())
-        .credential(new AzureKeyCredential(apiKey)).buildClient();
+RadiologyInsightsClientBuilder clientBuilder = new RadiologyInsightsClientBuilder()
+        .endpoint(endpoint).serviceVersion(RadiologyInsightsServiceVersion.getLatest());
+if (apiKey != null && !apiKey.equals("")) {
+    clientBuilder = clientBuilder.credential(new AzureKeyCredential(apiKey));
```

Comment:
Does service support AAD? If yes, we at least want to provide sample on that (you know, the Security initiative...).

Reply:
I added an extra sample that uses DefaultAzureCredential, which supports other ways of Authentication (managed identity, etc.)